styleguide/source/assets/scss/03-organisms/_wayfinder.scss

```diff
@@ -0,0 +1,42 @@
+.ama_wayfinder {
+  border-bottom: solid 1px $gray-8;
+
+  &--grid {
+    display: none;
+    padding: $gutter-mobile 0 $gutter-mobile ($gutter-mobile / 4);
+
+    @include breakpoint($bp-med) {
+      padding: ($gutter-mobile / 4) 0;
+    }
+    @include breakpoint($bp-small) {
+      display: flex;
+      flex-direction: row;
+    }
+  }
+
+  &--logo {
+    align-items: center;
+    display: flex;
+    flex-basis: 50%;
+  }
+
+  &--referrer {
+    align-self: center;
+    flex-basis: 50%;
+    display: flex;
+
+    &--link-back a {
+      display: none;
+      justify-content: center;
+      font-size: 0.9em;
+    }
+
+    &--link-back svg {
+      transform: rotate(90deg) translateY(-4px) translateX(7px); // adjust the positioning of the icon so it lines up with the text and the container edge.
+    }
+
+    &--link-back_text {
+      padding-left: 4px; // add some visual breathing room between the icon and the text.
+    }
+  }
+}
```

Comment:
All classes that are updated in the HTML will need to be updated in the SCSS as well. Also, please add an empty line at the end of the file.